src/networkAddresses.ts

```diff
@@ -86,6 +89,9 @@ export const NETWORK_ADDRESSES = {
     XVS_VAULT_PROXY: sepoliaDeployedContracts.addresses.XVSVaultProxy,
     GENERIC_TEST_USER_ACCOUNT: "0x2Ce1d0ffD7E869D9DF33e28552b12DdDed326706",
     WETH: "0x7b79995e5f793A07Bc00c21412e50Ecae098E7f9",
+    ENDPOINT: "0xae92d5aD7583AD66E49A0c67BAd18F6ba52dDDc1",
+    LZ_LIBRARY: "0x3acaaf60502791d199a5a5f0b173d78229ebfe32",
```

Comment:
I wasn't able to locate the addresses for the LZ_LIBRARY on testnets. Where did you get these addresses from?

Reply:
The LZ_Library address can be obtained by accessing [this](https://sepolia.etherscan.io/address/0xae92d5aD7583AD66E49A0c67BAd18F6ba52dDDc1#readContract#F13) function of the Endpoint.